src/js/main.js

```diff
@@ -154,7 +154,7 @@ fetch('./src/data/STREAM_GAGES_USED.json')
   .then(geojson => {
     const getInfo = (feature, layer) => {
         layer.bindTooltip('USGS Stream Gage', {permanent: true, direction: 'bottom', offset: [0,10], className: 'usgs-stream-gages-tooltip'})
-        layer.bindPopup(`<span align="center"><b>${feature.properties.AGENCY}</b> <br>Stream Gage #: ${feature.properties.GAGE_NUMBE} <br><i>${feature.properties.NAME}</i></span>`);
+        layer.bindPopup(`<span align="center"><b>${feature.properties.AGENCY}</b> <br>Stream Gage #: ${feature.properties.GAGE_NUMBE} <br><i><a href=${feature.properties.USGS_link} >${feature.properties.NAME}</i></a></span>`);

```

Comment:
@zapatam671 Can you make the link open in a new tab? Currently, it opens in the same tab as the map

Reply:
I have made the changes and pushed a commit into develop2